docs-kits/kits/Digital Twin Kit/page_changelog.md

```diff
@@ -0,0 +1,24 @@
+---
+id: Digital Twin Kit Changelog
+title: Changelog
+description: 'Digital Twin Kit'
+sidebar_position: 1
+---
+
+### Digital Twin Kit
+
+All notable changes to this Kit will be documented in this file.
+
+## [1.0.0] - 2023-04-12
+
+### Added
+
+- Initial version of the Kit including adoption, operation and development view + all relevant API specifications.
```

Comment:
Kit or KIT

Reply:
Initial version of the Kit including adoption, operation and development view, and all relevant API specifications.